pkg/analyze/host_subnetavailable.go

```diff
@@ -0,0 +1,82 @@
+package analyzer
+
+import (
+	"encoding/json"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/replicatedhq/troubleshoot/pkg/collect"
+)
+
+type AnalyzeHostSubnetAvailable struct {
+	hostAnalyzer *troubleshootv1beta2.SubnetAvailableAnalyze
+}
+
+func (a *AnalyzeHostSubnetAvailable) Title() string {
+	return hostAnalyzerTitleOrDefault(a.hostAnalyzer.AnalyzeMeta, "Subnet Available")
+}
+
+func (a *AnalyzeHostSubnetAvailable) IsExcluded() (bool, error) {
+	return isExcluded(a.hostAnalyzer.Exclude)
+}
+
+func (a *AnalyzeHostSubnetAvailable) Analyze(getCollectedFileContents func(string) ([]byte, error)) ([]*AnalyzeResult, error) {
+	hostAnalyzer := a.hostAnalyzer
+
+	name := filepath.Join("host-collectors/subnetAvailable", "result.json")
```

Comment:
`"host-collectors/subnetAvailable"` -> `"host-collectors/subnet-vailable"`

Path names in the support bundle have followed this format

Reply:
It looks like they've all followed the `camelCase` approach for the collector name itself...?

```
~/git/replicatedhq/troubleshoot/pkg/analyze cpuid-984 *6 ❯ grep host-collectors *                                                                                                                        
grep: files: Is a directory
host_certificate.go:	name := filepath.Join("host-collectors/certificate", collectorName+".json")
host_disk_usage.go:	name := filepath.Join("host-collectors/diskUsage", "diskUsage.json")
host_disk_usage.go:		name = filepath.Join("host-collectors/diskUsage", hostAnalyzer.CollectorName+".json")
host_filesystem_performance.go:	name := filepath.Join("host-collectors/filesystemPerformance", collectorName+".json")
host_http.go:	name := filepath.Join("host-collectors/http", "result.json")
host_http.go:		name = filepath.Join("host-collectors/http", hostAnalyzer.CollectorName+".json")
host_httploadbalancer.go:	fullPath := path.Join("host-collectors/httpLoadBalancer", fmt.Sprintf("%s.json", collectorName))
host_subnetavailable.go:	name := filepath.Join("host-collectors/subnetAvailable", "result.json")
host_subnetavailable.go:		name = filepath.Join("host-collectors/subnetAvailable", hostAnalyzer.CollectorName+".json")
host_system_packages.go:	packagesFileName := "host-collectors/system/packages.json"
host_system_packages.go:		packagesFileName = fmt.Sprintf("host-collectors/system/%s-packages.json", a.hostAnalyzer.CollectorName)
host_tcp_connect.go:	fullPath := filepath.Join("host-collectors/connect", collectorName+".json")
host_tcploadbalancer.go:	fullPath := path.Join("host-collectors/tcpLoadBalancer", fmt.Sprintf("%s.json", collectorName))
host_tcpportstatus.go:	fullPath := path.Join("host-collectors/tcpPortStatus", "tcpPortStatus.json")
host_tcpportstatus.go:		fullPath = path.Join("host-collectors/tcpPortStatus", fmt.Sprintf("%s.json", hostAnalyzer.CollectorName))
host_udpportstatus.go:	fullPath := path.Join("host-collectors/udpPortStatus", "udpPortStatus.json")
host_udpportstatus.go:		fullPath = path.Join("host-collectors/udpPortStatus", fmt.Sprintf("%s.json", hostAnalyzer.CollectorName))
```

Reply:
Ah, yes, you are right.